Formula/goproxy.rb

```diff
@@ -25,8 +25,8 @@ def install
   test do
     bind_address = "127.0.0.1:#{free_port}"
     begin
-      server = IO.popen("#{bin}/goproxy -proxy=https://goproxy.io -listen=#{bind_address}", err: [:child, :out])
-      sleep 1
+      server = IO.popen("#{bin}/goproxy", "-proxy=https://goproxy.io", "-listen=#{bind_address}", err: [:child, :out])
```

Comment:
I think this needs to be:

```suggestion
      server = IO.popen(["#{bin}/goproxy", "-proxy=https://goproxy.io", "-listen=#{bind_address}"], err: [:child, :out])
```

because it takes a string `mode` as a second argument.

Though we should probably do a run without any changes to see if it is reproducible.

Reply:
I don't think it's reproducible when testing just this formula by itself, but: https://github.com/Homebrew/homebrew-core/actions/runs/1552430219

Reply:
Removing the `Process.kill` and `wait` stuff might be a way to make it happen. It'll either get auto-killed after 5 minutes or not at all.

Reply:
Ok, let's try that.

Reply:
Yes, it looks like omitting `Process.kill` and `Process.wait` causes the test to hang.

Reply:
Ah, but it still gets caught by the timeout.